_commands/basics/whoami.md

```diff
@@ -10,6 +10,7 @@ whoami
 ~~~ bash
 $ whoami
 mmdarden
+
```

Comment:
The newlines need to go before the line that has "~~~ bash" not the ending "~~~". Sorry for the confusion.